lib/DxilRootSignature/DxilRootSignature.cpp

```diff
@@ -190,6 +190,44 @@ void DeleteRootSignature(const DxilVersionedRootSignatureDesc * pRootSignature)
   delete pRootSignature;
 }
 
+template <typename RootSigDesc, typename RootParameterDesc>
+void ExtendRootSig(RootSigDesc &rootSigDesc, 
```

Comment:
Looks familiar. Was this meant to be consolidated?

Reply:
Sadly, this duplication was a down-side of keeping all the new/delete in the same place...

Reply:
With the change to allocate memory outside of DxilRootSignature.cpp, I can de-dupe this.